### DISCUSSION TITLE
3.9.4 not available for download

### DISCUSSION THREAD

**Discussion by khs1974** [SBFspot/SBFspot#616] [Q&A]
Hello,

Why is release 3.9.4 not available anymore for download?
This is the only version which worked for me te upload data from two SMA BT inverters simultaneously..
With the latest release this doens't work..

openhabian@openhabian:~ $ curl -s https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/sbfspot-config | sudo bash -s 3.9.4
SBFspot Configuration Tool V1.3.2
Running on Raspberry Pi 3 Model B Rev 1.2
Checking whiptail... Installed.
architecture=armhf
Raspbian GNU/Linux 11 (bullseye)
username=openhabian
userhome=/home/openhabian
Minimum SBFspot version supported by this tool is 3.5.0
Request to install version 3.9.4
**Version 3.9.4 is not available for installation.**
Select one of 3.9.6

**Comment by SBFspot** [comment]
Confirmed as [bug](https://github.com/SBFspot/sbfspot-config/issues/55)

**Reply by sbf-** [reply]
[Fixed](https://github.com/SBFspot/sbfspot-config/commit/421aaf3eb0fd01fabc14590a92c1d48242ecfb59)

**Reply by khs1974** [reply]
I can run the installation of version 3.9.4. now, but during the installation I get the error message: "Unable to untar sbfspot-sqlite-arm-linux-bullseye.tar.gz"

openhabian@openhabian:~ $ curl -s https://raw.githubusercontent.com/sbfspot/sbfspot-config/master/sbfspot-config | sudo bash -s 3.9.4
[sudo] password for openhabian:
SBFspot Configuration Tool V1.3.3
Running on Raspberry Pi 3 Model B Rev 1.2
Checking whiptail... Installed.
architecture=armhf
Raspbian GNU/Linux 11 (bullseye)
username=openhabian
userhome=/home/openhabian
Minimum SBFspot version supported by this tool is 3.5.0
Request to install version 3.9.4
Searching for existing configuration(s)...
Reading settings from /usr/local/bin/sbfspot.3/SBFspot.cfg
Reading settings from /usr/local/bin/sbfspot.3/SBFspotUpload.cfg
Saving defaults to /var/tmp/sbfspot.3/sbfspot.defaults... Done
Performing some checks before installation... Done
Downloading sbfspot-sqlite-arm-linux-bullseye.tar.gz
tar: sbfspot-sqlite-arm-linux-bullseye.tar.gz: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
Unable to untar sbfspot-sqlite-arm-linux-bullseye.tar.gz
Installation aborted

**Reply by sbf-** [reply]
[Fixed](https://github.com/SBFspot/sbfspot-config/issues/57)

**Reply by khs1974** [reply]
everything works again, thanks!!!!
